### PR TITLE
add book reference at the end

### PR DESCRIPTION
**Pull request by lvwerra**
This PR adds a more explicit reference to the book at the end of the CodeParrot blog post.

### CODE REVIEW

File: codeparrot.md [hunk 1]
```diff
@@ -403,4 +403,6 @@ pipe('def hello_world():')
 
 ## Summary
 
-In this short blog post we walked through all the steps involved for training a large GPT-2 model called CodeParrot 🦜 for code generation. Using 🤗 Accelerate we built a training script with less than 200 lines of code that we can effortlessly scale across many GPUs. With that you can now train your own GPT-2 model!
\ No newline at end of file
+In this short blog post we walked through all the steps involved for training a large GPT-2 model called CodeParrot 🦜 for code generation. Using 🤗 Accelerate we built a training script with less than 200 lines of code that we can effortlessly scale across many GPUs. With that you can now train your own GPT-2 model!
+
+This post gives a brief overview of CodeParrot :parrot:, but if you are interested in diving deeper into how we pretrained this model we recommend reading its dedicated chapter in the upcoming [book on Transformers and NLP](https://learning.oreilly.com/library/view/natural-language-processing/9781098103231/).  This chapter provides many more details around building custom datasets, design considerations when training a new tokenizer, and architecture choice.
```
Comment (osanseviero) [inline codeparrot.md:408]:
```suggestion
This post gives a brief overview of CodeParrot 🦜, but if you are interested in diving deeper into how we pretrained this model, we recommend reading its dedicated chapter in the upcoming [book on Transformers and NLP](https://learning.oreilly.com/library/view/natural-language-processing/9781098103231/). This chapter provides many more details around building custom datasets, design considerations when training a new tokenizer, and architecture choice.
```

**Review by osanseviero — APPROVED** [review]
LG! Left some minor suggestions

**Review by lewtun — APPROVED** [review]
LGTM 🦜 🦜 🦜 !

File: codeparrot.md (hunk 1 above)
Comment (lewtun) [reply codeparrot.md:408]: Now that I can see the full context, perhaps we should also fix the tense from present to past?

E.g.

> ... train your own GPT-2 model! 
>
> If you are interested in diving deeper into how we pretrained CodeParrot 🦜, we recommend reading ...

WDYT?

**Pull request merged**